README.md

```diff
@@ -1,5 +1,5 @@
 <!-- @format -->
-> # Black Lives Matter 🖤 Free Palestine 🇵🇸
+> # Black Lives Matter 🖤 Free Palestine from Hamas 🇵🇸
```

Comment:
Why it doesn't matter from who to free it? 🙂  If you're really care that doesn't bother you. Besides that, If it was `#isrealUnderAttack` with the flag of Israel then you'll likely write "Thanks for your kind words" and send love emojis.. So let's be honest with each other

Reply:
He didn't say it should be replaced with Israel Under Attack.

Many GraphQL libraries were created and maintained by Israelis but you don't see them making political statements on the repos.

It matters who it's being saved from because without context it implies Israel is the culprit rather than the Hamas terrorist regime in charge of Gaza.

Ideally nothing would be written at all. But if you need to take a stance at least make that stance against terrorism.

Reply:
I personally do think that Gaza is under siege both from Israel and Egypt and that's a fact. Hamas is only a side-effect of living in this conditions. Besides that, Israel started killing and throwing Palestinians from their homes even before Hamas was created.. But that's an argument that we can't close since decades and it's a waste of time to go into the details.

I personally won't do such a change with political opinion on my repos and thanks for the Israelis/non Israelis developers for their contributions.

Reply:
Yes, before Hamas Israel had to deal with other Palestinian terrorists and wars.

Gaza is not under siege. Israel moved out of Gaza in 2005 with the hope of peace. Instead Hamas decided to turn it into a military base. Israel has had tens of thousands of missiles fired at it since.

There wouldn't be any blockade on Gaza if it were peaceful. Dialog is important. I understand everyone has ingrained beliefs but not closing down dialog people will never be able to see eye to eye.

I realise GitHub isn't the place for this. But that's the point. This should have never have entered the repo to begin with.

Reply:
"I **personally** do think that Gaza is under siege both from Israel and Egypt and that's a **fact**." - where do you think Gaza gets their food and other merchants if not from Israel?? there is only a siege because if their will be open port and airport in Gaza, then Hamas will import missiles that can harm Israel even more, and then all the Arabs, Muslims and the left media will call out Israel for defending itself. This way the conflict will never end. It unfortunate that this discussion is made here. Back to your statement "Why it matters from who to free it If you really care? " It does matter, cause for most of the people who chant this: Free Palestine means the destruction of Israel, you know: "from the river to the sea Palestine would be free", it's a clear call for the destruction of the jewish state  and that's a FACT. and that's why people here has a problem with that.

Reply:
That's naive, and you know it. 

> Why it matters from who to free it If you really care?

We all know what's being meant by that. In the eyes of "Palestine", the entity it needs to be freed from is the conquering force. It's used in context of anti-Israel protests, etc. You can't just ignore the context these words generate and treat them literally. That's just not what those words mean.

> Besides that, If it was #isrealUnderAttack with the flag of Israel then you'll likely write "Thanks for your kind words" and send love emojis and you'll not say "I don't think GitHub is an appropriate place for such political statements".. 

I would. I absolutely would. And if you show me to a project where that happened I would happily open an issue there and request removal. I think it has no room here, in this place of code and rational thinking. Politics is the death of any rational thought, and that attitude is not conducive to any sort of real engineering.

> Also, putting Palestine flag (with no humiliation to Israel flag) shouldn't bother you that much 🤷‍♂️

Again, context matters. That flag isn't really used for harmony peace and unity, it's used as a symbol of de-legitimizing the existence of Israel as a state. It's used in exclusion-based discussion. Putting that flag up (alongside the provocative text) makes it rather obvious what the message being conveyed here. The fact that the maintainer has not deigned to respond to this or any of the other issues/PRs further cements the fact that this is the intended message.

You wanted honesty, don't rationalize an obvious message and try to make it mean something it does not. I, personally, disagree with the message in some levels, and agree with it in others. But I still think it is **completely inappropriate** in the context of open source software, and code in general.

Reply:
I'm leaving the project on principle, but I think it would be best for the other maintainers moving forward to have any statement about Palestine or Israel removed.

I removed myself as a maintainer on principle, so I don't have power to make any changes

Reply:
What's the reason you decided to add it, then remove yourself from the project, and now recommend someone else removes it from the README?

Reply:
Oded, Why you've the right to fight against anyone that disagrees with Israel as a state (disclaimer: no Jewish is mentioned here) but you don't Palestinians to fight what has been eating their land and attacking them since years? That's a double standards.

Madara, I agree with most of your reply and I said that I personally wouldn't put political statements in my repos

Reply:
Oded didn't say anything like that.

Israel had the right to defend itself. Hamas doesn't have the right to fire thousands of rockets at civilians and call for the destruction of Israel.

There's no double standard. You can run all the political campaigns you want and basically everyone agrees OSS projects is not the place for that.

Reply:
Each side will give all the rights to do whatever they want in the name of anything.. unsubscribing from this thread ✅

Reply:
It's sad that you support terrorism in that case and see everything as morally equivalent.

Attacking civilians is wrong. Protecting civilians isn't. Whatever you think about the larger issues involved.